From e86f4a88cca1f4140ef5ee233f90c02ec1e44ee8 Mon Sep 17 00:00:00 2001
From: Emma Green <emma.green.software@gmail.com>
Date: Tue, 6 Apr 2021 17:25:36 -0400
Subject: [PATCH 0421/1021] small changes to the way the price is displayed on
 the course home page to pave the way for the expiration box

---
 src/course-home/outline-tab/OutlineTab.test.jsx | 4 ++--
 src/generic/course-sock/CourseSock.test.jsx     | 2 +-
 src/generic/upgrade-button/FormattedPricing.jsx | 2 +-
 src/generic/upgrade-button/UpgradeButton.jsx    | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/course-home/outline-tab/OutlineTab.test.jsx b/src/course-home/outline-tab/OutlineTab.test.jsx
index e12c8d1..a2c1740 100644
--- a/src/course-home/outline-tab/OutlineTab.test.jsx
+++ b/src/course-home/outline-tab/OutlineTab.test.jsx
@@ -899,7 +899,7 @@ describe('Outline Tab', () => {
 
     it('displays link to upgrade', async () => {
       await fetchAndRender();
-      expect(screen.getByRole('link', { name: 'Upgrade ($149)' })).toBeInTheDocument();
+      expect(screen.getByRole('link', { name: 'Upgrade for $149' })).toBeInTheDocument();
     });
 
     it('viewing upgrade card sends analytics', async () => {
@@ -930,7 +930,7 @@ describe('Outline Tab', () => {
       // Clearing after render to remove any events sent on view (ex. 'Promotion Viewed')
       sendTrackEvent.mockClear();
       sendTrackingLogEvent.mockClear();
-      const upgradeButton = screen.getByRole('link', { name: 'Upgrade ($149)' });
+      const upgradeButton = screen.getByRole('link', { name: 'Upgrade for $149' });
 
       fireEvent.click(upgradeButton);
 
diff --git a/src/generic/course-sock/CourseSock.test.jsx b/src/generic/course-sock/CourseSock.test.jsx
index 585e262..a70a088 100644
--- a/src/generic/course-sock/CourseSock.test.jsx
+++ b/src/generic/course-sock/CourseSock.test.jsx
@@ -36,7 +36,7 @@ describe('Course Sock', () => {
 
     expect(screen.getByText('edX Verified Certificate')).toBeInTheDocument();
     const { currencySymbol, price } = mockData.verifiedMode;
-    expect(screen.getByText(`Upgrade (${currencySymbol}${price})`)).toBeInTheDocument();
+    expect(screen.getByText(`Upgrade for ${currencySymbol}${price}`)).toBeInTheDocument();
 
     fireEvent.click(upsellButton);
     expect(screen.queryByText('edX Verified Certificate')).not.toBeInTheDocument();
diff --git a/src/generic/upgrade-button/FormattedPricing.jsx b/src/generic/upgrade-button/FormattedPricing.jsx
index d5d341e..a1e061f 100644
--- a/src/generic/upgrade-button/FormattedPricing.jsx
+++ b/src/generic/upgrade-button/FormattedPricing.jsx
@@ -49,7 +49,7 @@ function FormattedPricing(props) {
         {intl.formatMessage(messages.srPrices, { discountedPrice, originalPrice })}
       </span>
       <span aria-hidden="true">
-        <span>{discountedPrice}</span> <del>{originalPrice}</del>
+        <span>{discountedPrice}</span> (<del>{originalPrice}</del>)
       </span>
     </>
   );
diff --git a/src/generic/upgrade-button/UpgradeButton.jsx b/src/generic/upgrade-button/UpgradeButton.jsx
index a347a6a..84ff94e 100644
--- a/src/generic/upgrade-button/UpgradeButton.jsx
+++ b/src/generic/upgrade-button/UpgradeButton.jsx
@@ -26,7 +26,7 @@ function UpgradeButton(props) {
     >
       <FormattedMessage
         id="learning.upgradeButton.buttonText"
-        defaultMessage="Upgrade ({pricing})"
+        defaultMessage="Upgrade for {pricing}"
         values={{
           pricing: (
             <FormattedPricing
-- 
2.34.1

